scripts/start-travis-elasticsearch.sh

```diff
@@ -4,12 +4,9 @@ set -ex
 
 # Why download it and not use the elasticsearch service?
 # Two reasons
-#  - predictable version (1.4.x) is what we use in production
+#  - predictable version (5.1.x) is what we use in production
 #  - running it as a service would require sudo which we don't want to use
-wget --no-check-certificate 'https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.4.5.tar.gz'
-tar zxf elasticsearch-1.4.5.tar.gz
-echo "script.disable_dynamic: false" > elasticsearch.yml
-echo "index.number_of_shards: 1" >> elasticsearch.yml
-echo "index.number_of_replicas: 0" >> elasticsearch.yml
-# the -d flag starts it as a daemon in the background
-./elasticsearch-1.4.5/bin/elasticsearch -d --config=elasticsearch.yml
+wget --no-check-certificate 'https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.1.1.deb'
+sudo dpkg -i --force-confnew elasticsearch-5.1.1.deb
+
+sudo service elasticsearch start
```

Comment:
I think we're fine using the default values. Anyway, we can't set the config options we use to anymore, because ES won't accept those values in the yml file.